src/applications/vaos/components/AddToCalendar.jsx

```diff
@@ -9,11 +13,11 @@ import guid from 'simple-guid';
  * 
  * Additionally, any actual line breaks in the text need to be escaped
  */
-const ICS_LINE_LIMIT = 74;
+export const ICS_LINE_LIMIT = 74;
```

Comment:
Is this only used in tests? Might be good to move it somewhere else if so, similar to my comment below about the tokens function.